src/catkin_pkg/package.py

```diff
@@ -137,6 +137,18 @@ def has_buildtool_depend_on_catkin(self):
         """
         return 'catkin' in [d.name for d in self.buildtool_depends]
 
+    def build_type(self):
+        """
+        Returns value of export/build_type element, or 'catkin' if unspecified.
+
+        :returns: package build type
+        :rtype: str
+        """
+        for e in self.exports:
+            if e.tagname == 'build_type':
+                return e.content
+        return 'catkin'
```

Comment:
@dirk-thomas @wjwwood PR updated with a `build_type` check instead of buildtool. Could make this a property rather than a method by adding it to `getattr`, up to you guys if you'd prefer it there.